ocfweb/docs/docs/staff/mailing-lists.md

```diff
@@ -0,0 +1,53 @@
+[[!meta title="Staff mailing lists"]]
+
+## Mailing lists
+
+There are several mailing lists used internally by staff. Append
+`@ocf.berkeley.edu` to the end of each mailing list.
+
+All staffers are automatically added to the following two mailing lists:
+
+ * `staff`: For staff announcements, such as meeting times and events.
+ * `wheel`: For technical discussion among all staff
+
+Staffers can also choose to be further added to the following mailing lists.
+Technical Managers are especially expected to join them:
```

Comment:
I'd make it a policy consideration that /root or /admin requires that you be CC'ed on rt@ and root@